[0m00:18:40.045743 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106ef55b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1123c2820>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1123c2850>]}


============================== 00:18:40.048464 | 731cb89d-4efa-4c2c-b260-925e6adc81b7 ==============================
[0m00:18:40.048464 [info ] [MainThread]: Running with dbt=1.7.2
[0m00:18:40.048818 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/Users/dawidpilarski/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m00:18:40.049148 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/dawidpilarski/.dbt')}
[0m00:18:40.049327 [info ] [MainThread]: Creating dbt configuration folder at 
[0m00:19:04.002243 [debug] [MainThread]: Starter project path: /opt/homebrew/lib/python3.9/site-packages/dbt/include/starter_project
[0m00:19:04.011164 [info ] [MainThread]: 
Your new dbt project "cosmosproject" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m00:19:04.011980 [info ] [MainThread]: Setting up your profile.
[0m00:19:04.013075 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m00:19:04.025498 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 23.989042, "process_user_time": 1.03602, "process_kernel_time": 0.071997, "process_mem_max_rss": "109314048", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m00:19:04.026751 [debug] [MainThread]: Command `dbt init` failed at 00:19:04.026506 after 23.99 seconds
[0m00:19:04.027439 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106ef55b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1123c2970>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1123c28b0>]}
[0m00:19:04.028112 [debug] [MainThread]: Flushing usage events
